src/views/AccessControl.tsx

```diff
@@ -325,13 +333,25 @@ export const AccessControl = () => {
                                             setCurrentPage(page)
                                         }
                                     }}
-                                    className={`${showTutorial ? "card-table card-table-no-placeholder" : "card-table"}`}
+                                    className={`access-control-table ${showTutorial ? "card-table card-table-no-placeholder" : "card-table"}`}
                                     showSorterTooltip={false}
                                     scroll={{x: true}}
                                     dataSource={dataTable}>
                                     <Column title="Name" dataIndex="Name"
                                             onFilter={(value: string | number | boolean, record) => (record as any).Name.includes(value)}
-                                            sorter={(a, b) => ((a as any).Name.localeCompare((b as any).Name))} />
+                                            sorter={(a, b) => ((a as any).Name.localeCompare((b as any).Name))}
+                                            render={(text, record, index) => {
+                                                const desc = (record as RuleDataTable).Description.trim()
+                                                return <Tooltip title={desc !== "" ?  desc : "no description"} arrowPointAtCenter>
+                                                    <span className="tooltip-label">{text}</span>
+                                                </Tooltip>
+                                            }}
+                                    />
+                                    <Column title="Status" dataIndex="Disabled"
```

Comment:
Codacy has a fix for the issue: [Replace `··················<Column·title="Status"` with `/>⏎··················<Column⏎····················title="Status"⏎···················`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
                  />
```